xml/System.Reflection.Metadata/CustomAttributeNamedArgument`1.xml

```diff
@@ -15,9 +15,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <typeparam name="TType">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="TType">The type of the argument.</typeparam>
```

Comment:
>The type of the argument [](start = 28, length = 24)

Type used to represent types of values decoded from the custom attribute signature.